okhttp-tests/src/test/java/okhttp3/CipherSuiteTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
I’d like to have copyright for contributions to Square projects be assigned to Square. If you’re uncomfortable with this, that’s okay, but I’m going to decline this PR. We can still implement this functionality, but I’ll ask somebody else to do so.